README.md

```diff
@@ -0,0 +1,128 @@
+# Mountsion CBS Secondary Website
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. Here's how you can get set up
+
+-First head to the repository, located at https://github.com/MountSionCBSSecondary/SchoolWebsite
+
+-Then clone our repostory
+
+![alt text](https://github.com/MountSionCBSSecondary/SchoolWebsite/tree/master/assets/images/Screenshot from 2018-04-17 12-12-37.png)
+
+First you're going to click on "Clone or Download"
+
+Then proceed to copy the link
+
+---
+
+Now open your terminal
+
+Type
+`git clone <url>`
+
+### Terminal
+
+Wherever you are in terminal is where your file will be saved. TO navigate through the terminal you will only need 3 simple commands
+
+`cd`
+
+`ls`
+
+`cd..`
+
+The `cd` command is for changing directories, it is used by typing `cd <name-of-file>`
+
+The `ls` command will show all the files in the current directory, this makes it easy to navigate between files
+
+the `cd..` command will go back a directory, if you accidentally go to the wrong file you can use this to go back
+
+### Prerequisites
+
+Some software that will improve production consist of
+
+```
+Brackets
+```
+
+```
+Sublime
+```
+
+```
+Terminator
+```
+
+### Installing
+
+A step by step series of examples that tell you have to get a development env running
+
+Say what the step will be
+
+```
+Give the example
+```
+
+And repeat
+
+```
+until finished
+```
+
+End with an example of getting some data out of the system or using it for a little demo
+
+## Running the tests
+
+Explain how to run the automated tests for this system
+
+### Break down into end to end tests
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+### And coding style tests
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+## Deployment
+
+Add additional notes about how to deploy this on a live system
+
+## Built With
+
+* [Dropwizard](http://www.dropwizard.io/1.0.2/docs/) - The web framework used
+* [Maven](https://maven.apache.org/) - Dependency Management
+* [ROME](https://rometools.github.io/rome/) - Used to generate RSS Feeds
```

Comment:
Are we using `Dropwizard`, `Maven` and `Rome`?

Reply:
Like I said above, this was from a template that Laura provided on the old pr